src/components/Tabber/Tabber.tsx

```diff
@@ -78,6 +79,7 @@ function Tabber(): ReactNode {
         </div>
         <div className={scss.previewContainer}>
           {tabs[currentTab].posts[tabs[currentTab].postModeOnView].text}
+          <Preview>Preview</Preview>
```

Comment:
No caso coloca a linha 81 como o conteúdo do preview 
```suggestion
          
          <Preview>{tabs[currentTab].posts[tabs[currentTab].postModeOnView].text}</Preview>
```